### PR TITLE
Update to allow different chunk_limit_size for kinesis

### PR DESCRIPTION
**Pull request by keperry**
This issue details why this PR was made.  https://github.com/fluent/fluentd-kubernetes-daemonset/issues/325

Allows the user to set different chunk_limit_size based on env var

### CODE REVIEW

**Comment by keperry** [comment]
@shiftky - could you please take a look at this guy?

File: templates/conf/fluent.conf.erb [hunk 1]
```diff
@@ -361,7 +361,7 @@
 <% if is_v1 %>
   <buffer>
     flush_interval 1
-    chunk_limit_size 1m
+    chunk_limit_size "#{ENV['FLUENT_KINESIS_STREAMS_CHUNK_LIMIT_SIZE'] || 1m}"
```
Comment (repeatedly) [inline templates/conf/fluent.conf.erb]: 1m should be '1m'

Comment (keperry) [reply templates/conf/fluent.conf.erb]: @repeatedly  - nice catch and updated!

**Comment by keperry** [comment]
@repeatedly or @ganmacs - Thanks for the help!  How do we go about getting this merged?  Not sure on the process after approval.

**Comment by repeatedly** [comment]
Sorry for the delay. Just merged!

**Pull request merged**